sixtyfps_compiler/generator/cpp.rs

```diff
@@ -634,21 +634,18 @@ fn generate_public_component(file: &mut File, component: &llr::PublicComponent)
     generate_item_tree(
         &mut component_struct,
         &component.item_tree,
-        &component,
+        component,
         None,
-        component_id.clone(),
+        component_id,
         Access::Private, // Hide properties and other fields from the C++ API
         file,
     );
 
     // Give generated sub-components, etc. access to our fields
 
     for new_decl in file.declarations.iter().skip(old_declarations) {
-        match new_decl {
-            Declaration::Struct(struc @ Struct { .. }) => {
-                component_struct.friends.push(struc.name.clone());
-            }
-            _ => {}
+        if let Declaration::Struct(struc @ Struct { .. }) = new_decl {
```

Comment:
```suggestion
        if let Declaration::Struct(struc) = new_decl {
```

Paradoxically, i think this stray @ has little value... I guess i'm not consistent